data/tabular/uniprot_binding_sites/transform.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+
+FILENAME = "uniprot_binding_sites"
+
+
+def load_dataset() -> pd.DataFrame:
+    uniprot = pd.read_csv(
+        f"https://huggingface.co/datasets/chemNLP/uniprot/resolve/main/{FILENAME}/data_clean.csv"  # noqa: E501
```

Comment:
does this work with private repos? How das pandas know about the authentication?

Reply:
at least I struggle, but maybe I do something wrong